plenum/common/messages/node_messages.py

```diff
@@ -282,10 +281,11 @@ class MessageReq(MessageBase):
     """
     Purpose: ask node for any message
     """
+    allowed_types = {LEDGER_STATUS, CONSISTENCY_PROOF, PREPREPARE,
```

Comment:
Please add a test for `MessageReq` and `MessageRep` in `plenum/test/input_validation/message_validation` similar to other tests in this package. The tests are needed to check the contract of the messages.